frontend/src/containers/CreateEditPost.tsx

```diff
@@ -47,11 +47,21 @@ function CreateEditPost(props: PropsType) {
   const [postInfoData, setPostInfoData] = useState<PostInfoDataType>(initialFolderData);
   const [locationQuery, setLocationQuery] = useState("");
   const [selectedDay, setSelectedDay] = useState(1);
+  const [routePlaces, setRoutePlaces] = useState<PlaceDayType[]>([]);
+
+  // const onAddPlace = (place: PlaceType) => {
```

Comment:
Please remove these lines they are unnecessary!

Reply:
I will remove comments and keep ```const [routePlaces, setRoutePlaces] = useState<PlaceDayType[]>([])``` since this function needs to be delivered to CreateCardPost component!